src/components/Proposal/ProposalSummary/ProposalSummaryPluginManager.tsx

```diff
@@ -128,6 +137,23 @@ class ProposalSummary extends React.Component<IProps, IState> {
                           <img className={css.copyButton} src="/assets/images/Icon/Copy-blue.svg" onClick={this.copyToClipboardHandler(proposalState.pluginToRegisterData)} />
                         </td>
                       </tr>
+
+                      { votingParams && (
```

Comment:
is there a need to add check of votingParams and/or decodedData is not `undefined` ?
this can happened if [this condition ](https://github.com/daostack/alchemy/pull/1884/files#diff-9fe300ece25e1d080948a828baef1b9fR59) is not met.

Reply:
Actually there is a checking for that. The line itself (141): 
`votingParams &&` ---> meaning if `votingParams` is defined continue and add more table rows.